brainlit/utils/session.py

```diff
@@ -196,6 +199,7 @@ def pull_voxel(
         vox_in_img = voxel - np.array(bounds.to_list()[:3])
         return np.squeeze(np.array(img)), bounds, vox_in_img
 
+    # TOMMY REVIEW
     def pull_vertex_list(
```

Comment:
in the docs you say pull a "region." HOwever i've seen other places where you use other words like "subvolume" or "image." Go ahead and decide on a single word and use only it.

Reply:
let's go with volume and subvolume